docs/DOCKER.md

```diff
@@ -1,11 +1,170 @@
+  oracle: connectors.sources.oracle:OracleDataSource
+  mssql: connectors.sources.mssql:MSSQLDataSource
+```
+
+Notice, that the config file you downloaded might contain more config entries, so you will need to manually copy/change the settings that apply to you. It should be sufficient to only update `elasticsearch.host` and `elasticsearch.password` to make Connectors Service run properly for you.
+
+### Running connector service in [native mode](https://www.elastic.co/guide/en/enterprise-search/current/native-connectors.html) in Docker
+
+In case you want to run connector service in native mode, you will need slightly different configuration file, having `native_service_types` config option populated instead of `connector_id` and `service_type`. Normally when you download a sample configuration file in step 2 you'll be able to run connector service in native mode (as long as elasticsearch host and credentials are correct).
+
+Example of configuration file for connectors service running in native mode:
+
+```
+elasticsearch:
+  host: http://host.docker.internal:9200
+  username: elastic
+  password: <YOUR_PASSWORD>
+  ssl: true
+  bulk:
+    queue_max_size: 1024
+    queue_max_mem_size: 25
+    display_every: 100
+    chunk_size: 1000
+    max_concurrency: 5
+    chunk_max_mem_size: 5
+    concurrent_downloads: 10
+  request_timeout: 120
+  max_wait_duration: 120
+  initial_backoff_duration: 1
+  backoff_multiplier: 2
+  log_level: info
+
+service:
+  idling: 30
+  heartbeat: 300
+  max_errors: 20
+  max_errors_span: 600
+  max_concurrent_syncs: 1
+  job_cleanup_interval: 300
+  log_level: INFO
+  
+# remove entries from this list to not run these connectors in this instance of the service
+native_service_types:
+  - mongodb
+  - mysql
+  - network_drive
+  - s3
+  - google_cloud_storage
+  - azure_blob_storage
+  - postgresql
+  - oracle
+  - dir
+  - mssql
+
+sources:
+  mongodb: connectors.sources.mongo:MongoDataSource
+  s3: connectors.sources.s3:S3DataSource
+  dir: connectors.sources.directory:DirectoryDataSource
+  mysql: connectors.sources.mysql:MySqlDataSource
+  network_drive: connectors.sources.network_drive:NASDataSource
+  google_cloud_storage: connectors.sources.google_cloud_storage:GoogleCloudStorageDataSource
+  azure_blob_storage: connectors.sources.azure_blob_storage:AzureBlobStorageDataSource
+  postgresql: connectors.sources.postgresql:PostgreSQLDataSource
+  oracle: connectors.sources.oracle:OracleDataSource
+  mssql: connectors.sources.mssql:MSSQLDataSource
+```
+
+After that, you can build your own docker image to run:
+
+```
+docker build -t <TAG_OF_THE_IMAGE> .
+```
+
+For example, if you've created a custom version of MongoDB connector, you can tag it with the following command:
+
+```
+docker build -t connector/custom-mongodb:1.0 .
+```
+
+You can later use `<TAG_OF_THE_IMAGE>` instead of `docker.elastic.co/enterprise-search/elastic-connectors:8.7.0.0-SNAPSHOT` in the next step to run the docker image
```

Comment:
```suggestion
You can later use `<TAG_OF_THE_IMAGE>` instead of `docker.elastic.co/enterprise-search/elastic-connectors:8.7.0.0-SNAPSHOT` in the next step to run the Docker image.
```